tests/integration/components/editor-with-preview-test.js

```diff
@@ -159,7 +159,7 @@ test('it sets the editor min-height to the editor height when previewing and sti
 
   this.$('.preview').click();
 
-  let style = 'min-height: ' + height + ';';
+  let style = `min-height: ${  height  };`;
```

Comment:
That is a little strange looking, again, probably a result of using --fix. I can hunt for weird spacing in the next commit.